core/modules/rproc/InfileMerger.cc

```diff
@@ -193,18 +159,43 @@ InfileMerger::InfileMerger(InfileMergerConfig const& c,
         return _deleteInvalidRows(jobAttempts);
     });
 
-    /* &&&INFILE
-    if (!_setupConnection()) {
-        throw InfileMergerError(util::ErrorCode::MYSQLCONNECT, "InfileMerger mysql connect failure.");
-    }
-    */
+
 }
 
 
 InfileMerger::~InfileMerger() {
 }
 
 
+void InfileMerger::_setEngineFromStr(std::string const& engineName) {
+    std::string eName;
+    for (auto&& c:engineName) { eName += toupper(c); }
+    if (eName == "INNODB") {
+        _dbEngine = INNODB;
+    } else if (eName == "MEMORY") {
+        _dbEngine = MEMORY;
+    } else if (eName == "MYISAM") {
+        _dbEngine = MYISAM;
+    } else {
+        LOGS(_log, LOG_LVL_ERROR, "unknown dbEngine=" << engineName << " using default MYISAM");
+        _dbEngine = MYISAM;
```

Comment:
An exception needs to be thrown here to tell users that Qserv is misconfigured, or there was a problem reading a configuration ile.